calendar/modules/listener/http_service.bal

```diff
@@ -13,61 +13,68 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+
 import ballerina/http;
 import ballerina/log;
 import ballerinax/googleapis.calendar;
 
-service class HttpService {
-
-    private boolean isOnNewEventAvailable = false;
-    private boolean isOnEventUpdateAvailable = false;
-    private boolean isOnEventDeleteAvailable = false;
-    private SimpleHttpService httpService;
-    private calendar:Client calendarClient;
+isolated service class HttpService {
+    private final boolean isOnNewEventAvailable;
+    private final boolean isOnEventUpdateAvailable;
+    private final boolean isOnEventDeleteAvailable;
+    private final HttpToCalendarAdaptor adaptor;
+    private final calendar:CalendarConfiguration & readonly calendarConfig;
     private string calendarId;
-    public string channelId;
-    public string resourceId;
+    private string channelId;
+    private string resourceId;
     private string? currentSyncToken = ();
 
-    public isolated function init(SimpleHttpService|HttpService httpService, calendar:Client calendarClient, 
-                                  string calendarId, string channelId, string resourceId) {
-        self.httpService = httpService;
-        self.calendarClient = calendarClient;
+    isolated function init(HttpToCalendarAdaptor adaptor, calendar:CalendarConfiguration config, string calendarId, 
```

Comment:
Seems this is a signature change in the connector?

Reply:
Yes but this is an internal change. This is not gonna break any sample or template.
For the user, this change is not known

Reply:
Understood. Should be fine. +1